consensus/polybft/polybft_config.go

```diff
@@ -64,10 +64,11 @@ func GetPolyBFTConfig(chainConfig *chain.Chain) (PolyBFTConfig, error) {
 
 // BridgeConfig is the rootchain bridge configuration
 type BridgeConfig struct {
-	BridgeAddr      types.Address `json:"stateSenderAddr"`
-	CheckpointAddr  types.Address `json:"checkpointAddr"`
-	AdminAddress    types.Address `json:"adminAddress"`
-	JSONRPCEndpoint string        `json:"jsonRPCEndpoint"`
+	BridgeAddr             types.Address `json:"stateSenderAddr"`
+	CheckpointAddr         types.Address `json:"checkpointAddr"`
+	RootERC20PredicateAddr types.Address `json:"rootERC20PredicateAddr"`
+	RootERC20Addr          types.Address `json:"rootERC20Addr"`
```

Comment:
Hope it is better this way. Wasn't sure about how to name it so that it is short enough and self-explanatory. :slightly_smiling_face: 
https://github.com/0xPolygon/polygon-edge/pull/1242/commits/06b992da38458ca3df75731995eacc950472894c.